style.css

```diff
@@ -37,39 +62,56 @@ input {
   box-sizing: border-box;
   border: none;
   border-radius: 4px 0 0 4px;
-  background: #e5e9ff;
-  color: #0026ff;
+  background: #8a89848c;
+  color: #ffffff;
   padding: 16px;
   font-size: 16px;
   font-family: 'Montserrat';
   font-weight: 600;
   line-height: 26px;
   flex: 1;
+  margin-top: 5%;
+}
+
+input:focus{
+
+outline: none;
+background: #888781e3;
 }
 
-main {
-  margin: 0 auto;
+
+.container__main {
+  position:absolute;
+  margin: 23vh 32vw;
   width: 100%;
   max-width: 700px;
   height: 600px;
   border-radius: 30px;
-  background: #fff;
   padding: 20px 24px;
   padding-top: 0;
   box-sizing: border-box;
   display: flex;
   flex-direction: column;
+  backdrop-filter: blur(20px);
 }
 
 .chat {
   flex: 1;
   display: flex;
   justify-content: flex-start;
   overflow: scroll;
+  overflow-x: hidden;
   flex-direction: column;
   padding-bottom: 16px;
 }
 
+::-webkit-scrollbar {
```

Comment:
What exactly does this webkit-scrollbar do?

Reply:
its remove the visibility of the scrollbar but keep the function of it :)

Reply:
same here. the scrollbar is still visible in firefox